app/models/beta/search/intercept_message.rb

```diff
@@ -1,12 +1,21 @@
 module Beta
   module Search
     class InterceptMessage
+      SECTION_REGEX = /(?<type>section)s? (?<roman_number>[XVI]{0,10})[.,\s)]/i
```

Comment:
I think we should support this as a number _and_ a roman number and probably also support optional position/id/code strings since sections are referred to in this way and the input is coming from HMRC/Matt/Sunny

Reply:
done!